internal/restic/id.go

```diff
@@ -83,6 +83,28 @@ func (id ID) Equal(other ID) bool {
 	return id == other
 }
 
+func (id ID) Less(other ID) bool {
```

Comment:
Isn't this just `bytes.Compare(id[:], other[:]) < 0`?

Reply:
In fact, the whole Equal/LessEqual/Equal machinery can be replaced by a single Compare function, whose implementation is trivial. That way, there's less chance of inconsistency between them.